yarn-project/simulator/src/avm/avm_context.test.ts

```diff
@@ -9,7 +9,8 @@ describe('Avm Context', () => {
 
     const newAddress = AztecAddress.random();
     const newCalldata = [new Fr(1), new Fr(2)];
-    const newContext = context.createNestedContractCallContext(newAddress, newCalldata);
+    const allocatedGas = { l1Gas: 1, l2Gas: 2, daGas: 3 };
```

Comment:
What are these numbers?

Reply:
It's how much of the current call gas we pass to the nested call. Added a comment to clarify.